units/vi/unit1/dummy-agent-library.mdx

```diff
@@ -337,8 +337,8 @@ Observation: The current weather in London is mostly cloudy with a high of 12°C
 ````
 
 Bạn thấy vấn đề chứ?
->Câu trả lời bị model hallucinate (ảo giác / tạo ra thông tin sai). Ta cần dừng lại để thực thi function thực sự!
-Giờ hãy dừng ở "Observation:" để không hallucinate kết quả function.
+>Mô hình đã bịa (hallucinate) ra câu trả lời. Ta cần dừng lại để thực thi function thực sự!
```

Comment:
IMO the word "hallucinate" is quite hard to be translate directly to vi, can we keep it and include an explanation?

Another way is "ảo giác", I've heard some of my friends say this word, but not sure if it is a common one nowadays.

Reply:
@honghanhh Could you please confirm if this is ok to change to "ảo giác"? Thanks.

Reply:
Updated 🚀